etl/steps/data/garden/irena/2023-12-12/renewable_power_generation_costs.meta.yml

```diff
@@ -1,46 +1,47 @@
-# TODO: This file is a duplicate of the previous step. It is not yet used in the dag and should be updated soon.
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Energy
+    short_unit: $/kWh
+    unit: 2022 US$ per kilowatt-hour
+    description_short: |-
+      Measured in 2022 US$ per kilowatt-hour.
+
+  description_key_lcoe: &description_key_lcoe
```

Comment:
on grapher it looks like the data was adjusted for inflation (I guess that's why it's in 2022 US dollars) - I'd include it in the description_key_lcoe as well